src/lib/vm-manager-hoc.jsx

```diff
@@ -54,12 +67,14 @@ const vmManagerHOC = function (WrappedComponent) {
         render () {
             const {
                 /* eslint-disable no-unused-vars */
+                cloudHost,
                 fontsLoaded,
                 loadingState,
                 onError: onErrorProp,
                 onLoadedProject: onLoadedProjectProp,
                 projectData,
                 projectId,
+                userName,
```

Comment:
I think almost everywhere else (including in the new CloudProvider class!) we use `username` without caps — can we stick with that for consistency?